apps/namadillo/src/registry/namadaAsset.ts

```diff
@@ -3,13 +3,14 @@ import namadaShieldedSvg from "./assets/namada-shielded.svg";
 
 export const namadaAsset: Asset = {
   name: "Namada",
-  base: "namnam",
+  base: "tnam1qy440ynh9fwrx8aewjvvmu38zxqgukgc259fzp6h",
```

Comment:
I think we cannot use this `tnam...` address because in the future it'll be replaced by the `namada-chain-registry`

I understand you explained that you will have another PR to support the housefire chain, which I believe you will update this again, but can you confirm this is on your plan before merging?

Reply:
Sorry, that's my bad that I didn't write the reason for this anywhere.

Since on Namada chains we need to do an extra step to decode the `tnam...` address to an asset, I added this here. Alternatively we can do this using the indexer, but this only works for the Namada chain configured in Namadillo. So if Namadillo is configured for the internal devnet, but our account contains some NAM originating in housefire, then we need a way to look up the `tnam...` addresses for housefire tokens, which means we either need an indexer for each Namada chain, or we can just store the address in the chain config, which I think is easier.

My plan is to add the tnam address to the Namada registry for each Namada chain so this will work for all Namada chains.

Reply:
We have the prop `address` for this case, maybe it's helpful. Do you mind giving it a try?

![Screenshot 2024-11-08 at 11 59 15](https://github.com/user-attachments/assets/25e682ea-4055-4981-aef8-c8058804545b)

Reply:
Oh nice! I didn't know about that. That's much better!

Reply:
I'm thinking I will merge this as is, and then next up is the chain registry integration, so I will fix the `tnam` thing there.